mysql-test/suite/innodb/t/percona_bug_ps5476.test

```diff
@@ -0,0 +1,4 @@
+call mtr.add_suppression("Encryption can't find master key, please check the keyring plugin is loaded.");
+call mtr.add_suppression("Can't set redo log tablespace to be encrypted.");
```

Comment:
Please add grep inc for these error messages.

Instead of number for test file names, it is better to use text. may be redo_log_encrypt_failure.test  or something like that..

Reply:
ofcourse dont forget percona_ prefix for our test :)